pyproject.toml

```
@@ -6,7 +6,8 @@ build-backend = "hatchling.build"
 name = "rpmautospec"
 version = "0.7.3"
 description = "A package and CLI tool to generate RPM release fields and changelogs."
-license = {text = "MIT"}
+license = "MIT AND (GPL-2.0-only WITH GCC-exception-2.0) AND (GPL-2.0-or-later WITH GCC-exception-2.0)"
```

The `(GPL-2.0-only WITH GCC-exception-2.0)` bit seems duplicated. Do you mean?

```suggestion
license = "MIT AND GPL-2.0-only WITH GCC-exception-2.0 AND (MIT OR GPL-2.0-or-later WITH GCC-exception-2.0)"
```

---

Yes, good catch! 👍

---

github still shows the original in the overall diff. this does not seem resolved

---

🫣 Forgot to commit & push. Should be fixed now!